.github/workflows/pull_request.yaml

```diff
@@ -44,7 +44,10 @@ jobs:
     - name: Setup containerd cluster
       run: |
         set -x
-        sudo apt-get install -y kubeadm kubelet
+        sudo curl -fsSLo /usr/share/keyrings/kubernetes-archive-keyring.gpg https://packages.cloud.google.com/apt/doc/apt-key.gpg
+        echo "deb [signed-by=/usr/share/keyrings/kubernetes-archive-keyring.gpg] https://apt.kubernetes.io/ kubernetes-xenial main" | sudo tee /etc/apt/sources.list.d/kubernetes.list
```

Comment:
are you worried about this failing in the future if you switch to `ubuntu-latest`?

Reply:
This is based on the official kube doc's

Following the breadcrumbs... https://apt.kubernetes.io/ --> https://packages.cloud.google.com/apt/dists 
```
...
kubernetes-jessie
kubernetes-jessie-unstable
kubernetes-lucid
kubernetes-lucid-unstable
kubernetes-precise
kubernetes-precise-unstable
kubernetes-squeeze
kubernetes-squeeze-unstable
kubernetes-stretch
kubernetes-stretch-unstable
kubernetes-trusty
kubernetes-trusty-unstable
kubernetes-wheezy
kubernetes-wheezy-unstable
kubernetes-xenial
kubernetes-xenial-unstable
kubernetes-yakkety
kubernetes-yakkety-unstable
...
```

It seems they're setting this up as Xenial as the LCD and don't have newer distros for ubuntu.

<br class="Apple-interchange-newline">